visualization/src/pcl_visualizer.cpp

```diff
@@ -2165,7 +2167,7 @@ void
 pcl::visualization::PCLVisualizer::resetCameraViewpoint (const std::string &id)
 {
   vtkSmartPointer<vtkMatrix4x4> camera_pose;
-  static CloudActorMap::iterator it = cloud_actor_map_->find (id);
+  CloudActorMap::iterator it = cloud_actor_map_->find (id);
```

Comment:
This is a behavioural change. The iterator is updated on every reset, not just the first. Please move to a different PR which can be marked as changes behavior (as well as fix (seems like a fix to me))